README.md

```diff
@@ -2,29 +2,32 @@
 lab manager for plate maps and sequence flows
 
 # Install
-Labman relies on the Qiita database. You will need first to install Qiita in
-a different environment (Qiita is Python 2 only, while labman is Python 3) and
-create the Qiita database using the [Qiita installation instructions](https://github.com/biocore/qiita/blob/master/INSTALL.md).  The instructions
+LabControl relies on the Qiita database and webserver. You will need first to 
+install Qiita in a different environment than LabControl (Qiita is Python 2 
+only, while LabControl is Python 3) using the [Qiita installation instructions](https://github.com/biocore/qiita/blob/master/INSTALL.md).  
+(It is also necessary to start the qiita webserver on port 8383, rather than the 
+default port for qiita of 21174; this can be done with the command 
+ `qiita pet webserver start --port 8383` .) The instructions
 below assume the Qiita PostgreSQL database is named `qiita_test`, which is the
 default name of the database created by the Qiita installation process; if your
 Qiita installation has a different database name, substitute that for
 `qiita_test` throughout.
 
-Once Qiita is installed, create a new, empty conda environment for labman.  
-Source this environment and install labman; start by first installing the
+Once Qiita is installed, create a new, empty conda environment for LabControl.  
```

Comment:
Is it 'LabControl' or 'Labcontrol'? I may need to revisit a few files. :)

Reply:
Customer-facing, it is canonically "LabControl": 
![Screen Shot 2019-04-17 at 1 05 01 PM](https://user-images.githubusercontent.com/10677935/56317596-f8526b00-6111-11e9-9092-109f18707ca0.png)

For developer-facing stuff (e.g., folder names, etc), it is fine for it to be "labcontrol".

Reply:
Gotcha, Already updated https://github.com/jdereus/labman/pull/459